salt/ext/win_inet_pton.py

```diff
@@ -22,7 +22,7 @@ class sockaddr(ctypes.Structure):
     WSAStringToAddressA = ctypes.windll.ws2_32.WSAStringToAddressA
     WSAAddressToStringA = ctypes.windll.ws2_32.WSAAddressToStringA
 else:
-    def not_windows():
+    def not_windows(*args):
```

Comment:
Does it need to accept args? What was the behavior of this function before (what was the bug)?